### PR TITLE
Add Nightly Blog theme

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

**Comment by MikkCZ** [comment]
Hi @kyoshino. I haven't noticed this PR before. What is the current status of it? May I help by giving it some pre-review, or is there some work yet to be done?

**Comment by kyoshino** [comment]
The theme itself has already been applied to https://blog.nightly.mozilla.org/ a while back. This PR just needs to be merged.

File: themes/frontierline-nightly/content-views/content.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Display a full post, no featured image.
```
Comment (MikkCZ) [inline themes/frontierline-nightly/content-views/content.php:3]: There is now option to hide the featured image in the parent theme. The only difference from the parent is the `#fx-footer` at the bottom.

File: themes/frontierline-nightly/content-views/content.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <?php the_title(); ?>
+      </a>
+    <?php else : ?>
+      <?php the_title(); ?>
+    <?php endif; ?>
+    </h1>
+
+    <div class="entry-info">
+    <?php if ($post->post_type === 'post') : ?>
+      <address class="vcard">
```
Comment (MikkCZ) [inline themes/frontierline-nightly/content-views/content.php:34]: The vcard is now configurable in the parent theme (`frontierline_no_byline` option). We should add it here too.

File: themes/frontierline-nightly/includes/nav-global.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?php
```
Comment (MikkCZ) [inline themes/frontierline-nightly/includes/nav-global.php:1]: I guess I should cover this in #13 too, after this is merged.

**Review by MikkCZ — COMMENTED** [review]
The `content.php` file should be updated to match the latest changes in the parent them, either now in the PR or after it's merged (I can definitely do that). @craigcook what do you think?

I will take care of the `nav-global.php` of course - the related change is still in a standalone PR.

**Review by craigcook — APPROVED** [review]

**Pull request merged**